src/main/java/fruitshop/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,23 @@
+package fruitshop.service.impl;
+
+import fruitshop.service.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String file) {
+        List<String> data = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(file))) {
+            String value = bufferedReader.readLine(); // skip header (first line)
+            while ((value = bufferedReader.readLine()) != null) {
+                data.add(value);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file " + file, e);
+        }
+        return data;
```

Comment:
According to the checklist, avoid hardcoding file paths in your services. You should pass the file path as a parameter to the method, which is done correctly here. However, ensure that the file path is not hardcoded elsewhere in the application, such as in the `Main` class or test classes.